input_values/dockers.json

```diff
@@ -1,25 +1,27 @@
 {
-  "name" : "dockers",
-  "cnmops_docker" : "us.gcr.io/broad-dsde-methods/vjalili/cnmops:lint-24b9cda",
-  "condense_counts_docker" : "us.gcr.io/broad-dsde-methods/gatk:condensecounts-7396ae99aaab07e29c92b509a6515508fbe68158",
-  "delly_docker" : "us.gcr.io/broad-dsde-methods/delly:8645aa",
-  "gatk_docker" : "us.gcr.io/broad-gatk/gatk:4.1.8.1",
-  "gatk_docker_pesr_override" : "us.gcr.io/broad-gatk/gatk:4.1.8.1",
-  "genomes_in_the_cloud_docker" : "us.gcr.io/broad-gotc-prod/genomes-in-the-cloud:2.3.2-1510681135",
-  "linux_docker" : "marketplace.gcr.io/google/ubuntu1804",
-  "manta_docker" : "us.gcr.io/broad-dsde-methods/manta:8645aa",
-  "melt_docker" : "us.gcr.io/talkowski-sv-gnomad/melt:vj-4ff9de9f",
-  "samtools_cloud_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/samtools-cloud:mw-gnomad-02-6a66c96",
-  "sv_base_docker" : "us.gcr.io/broad-dsde-methods/markw/sv-base:mw-gnomad-0506-pr-087d4df",
-  "sv_base_mini_docker" : "us.gcr.io/broad-dsde-methods/markw/sv-base-mini:mw-gnomad-0506-pr-087d4df",
-  "sv_pipeline_base_docker" : "us.gcr.io/broad-dsde-methods/markw/sv-pipeline-base:mw-gnomad-0506-pr-087d4df",
-  "sv_pipeline_docker" : "us.gcr.io/broad-dsde-methods/eph/sv-pipeline:eph_hotfix_no_evidence-1f461ed",
-  "sv_pipeline_qc_docker" : "us.gcr.io/broad-dsde-methods/markw/sv-pipeline-qc:mw-gnomad-0506-pr-087d4df",
-  "sv_pipeline_rdtest_docker" : "us.gcr.io/broad-dsde-methods/markw/sv-pipeline-rdtest:mw-gnomad-0506-pr-087d4df",
-  "wham_docker" : "us.gcr.io/broad-dsde-methods/wham:8645aa",
+  "name": "dockers",
+  "cnmops_docker": "us.gcr.io/broad-dsde-methods/tbrookin/cnmops:25a7945",
+  "condense_counts_docker": "us.gcr.io/broad-dsde-methods/gatk:condensecounts-7396ae99aaab07e29c92b509a6515508fbe68158",
+  "delly_docker": "us.gcr.io/broad-dsde-methods/delly:8645aa",
+  "gatk_docker": "us.gcr.io/broad-dsde-methods/gatk:condensecounts-7396ae99aaab07e29c92b509a6515508fbe68158",
+  "gatk_docker_pesr_override": "us.gcr.io/broad-dsde-methods/gatk:condensecounts-7396ae99aaab07e29c92b509a6515508fbe68158",
```

Comment:
I was told to only have build_docker.py manage the images that it has traditionally managed. These images in particular are "special sauce" images that are on forks that (to my knowledge) are not being fully maintained. I would be open to adding maintenance of images from gatk branches as an additional PR. It could be done straightforwardly via brute force (i.e. with lots of code duplication in independent Dockerfiles) or by adding additional functionality. I think it would be outside of the scope of this PR though.

Reply:
It was not clear to me at first, thanks for the clarification. In that case, I agree it would be better to keep them out of this PR.